integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/EnvironmentEditTest.java

```diff
@@ -50,39 +51,37 @@ protected void setupTest(TestContext testContext) {
             when = "change managed ssh key to existing one",
             then = "delete managed ssh key but not create new one")
     public void authenticationEditWhenSetExistingKeyAndDeleteManagedSuccessfully(MockedTestContext testContext) {
+        String randomPublicKeyId = UUID.randomUUID().toString();
         testContext
-                .given(HttpMock.class).whenRequested(SpiEndpoints.RegisterPublicKey.class).post()
-                .thenReturn((s, uriParameters) -> "")
-                .whenRequested(SpiEndpoints.GetPublicKey.class).get()
-                .pathVariable("publicKeyId", "id")
-                .thenReturn((s, uriParameters) -> true)
+                .given(HttpMock.class)
+                .mockSpi().getPublicKey().get()
```

Comment:
Honestly I am missing the original keywords here (given when then) a bit.
would it make sense call this as 'whenMockSpiRequestedWith()' or something similar?